bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewShadeHandler.java

```diff
@@ -619,8 +621,11 @@ private void removeListProcessChannel(List<Channel> removeList, String channelId
 
     /**
      * Remove previously statically created channels if the shade does not support them.
```

Comment:
Since having this Javadoc, to be correct, this line would now need to be rephrased slightly. The power/battery channels are supported for all power source configurations, so they are removed when **irrelevant**, not only when **unsupported**.
```suggestion
     * Remove previously statically created channels if the shade does not support them or they are not relevant.
```

Reply:
Ok.